source/extensions/filters/udp/dns_filter/dns_filter.h

```diff
@@ -42,24 +42,44 @@ using DnsVirtualDomainConfig = absl::flat_hash_map<std::string, AddressConstPtrV
 /**
  * DnsFilter configuration class abstracting access to data necessary for the filter's operation
  */
-class DnsFilterEnvoyConfig {
+class DnsFilterEnvoyConfig : public Logger::Loggable<Logger::Id::filter> {
```

Comment:
Is there a DnsFilter ID? Or do all filters use the same logger id?

Reply:
It's derived from the filter name IIRC.   I didn't have to do anything special here, but grepping for "dns_filter" in the logs will allow you to filter out the messages produced.